.gitignore

```diff
@@ -0,0 +1,3 @@
+gnore PHPUnit Test vendor scripts & environment info.
```

Comment:
I think this should be a comment: `# Gitignore PHPUnit Test vendor scripts & environment info.`

Reply:
Good catch. Totally looks like I got bit by vim 😛